test/specs/components/Component/Component-test.ts

```diff
@@ -0,0 +1,33 @@
+import { mountWithProvider as mount } from 'test/utils'
+
+import Component from 'src/components/Component/Component'
+import { isConformant } from 'test/specs/commonTests'
+
+describe('Component', () => {
+  const createComponentComp = (factoryFn: Function, val, options?) =>
+    mount(factoryFn(val, options)).find(Component)
+
+  xdescribe('is conformant', () => {
+    isConformant(Component)
+  })
+
+  describe(`create`, () => {
+    it(`renders a ${Component.defaultProps.as} element with children prop`, () => {
+      const testChildren = 'test children'
+      const component = createComponentComp(Component.create, testChildren)
+
+      expect(component.prop('as')).toEqual(Component.defaultProps.as)
+      expect(component.prop('children')).toEqual(testChildren)
+    })
+
+    it(`renders an HTML element with correct prop`, () => {
```

Comment:
I believe the `as` prop is covered in the conformance test.

Reply:
will remove, thanks